### DISCUSSION TITLE
Restore notifctions setting

### DISCUSSION THREAD

**Discussion by idan-green** [theyosh/TerrariumPI#520] [Q&A] [answered]
Hi,

When i tried to setup Telegram to send notifications.
During this time the systems asked me to change the messages which I did.
Now all the setting for notations are rest and don't work.
Can someone tell me how to restore the default setting?
![image](https://user-images.githubusercontent.com/41910075/107147262-b5503000-6955-11eb-856b-69a8aaaee138.png)

**Comment by theyosh** [comment, marked as answer]
Looks related to https://github.com/theyosh/TerrariumPI/issues/518
So do `git pull` and restart. Make sure your browser cache is cleaned
